### ISSUE TITLE
"find_blob_index_by_name data failed,  about layer BatchNorm" when load bin and params

### ISSUE THREAD

**Issue by xugangtongji** [Tencent/ncnn#995]

**Comment by xugangtongji** [comment]
![image](https://user-images.githubusercontent.com/26272437/57909483-7c387800-78b5-11e9-9636-00b942f2844e.png)

**Comment by xugangtongji** [comment]
the ncnn model is transferred by pytorch

**Comment by WenmuZhou** [comment]
mark

**Comment by HLIG** [comment]
mark
